Modules/FindMPFR.cmake

```diff
@@ -0,0 +1,23 @@
+# Try to find the GMP librairies
+#  MPFR_FOUND - system has GMP lib
+#  MPFR_INCLUDE_DIR - the GMP include directory
+#  MPFR_LIBRARIES - Libraries needed to use GMP
+
+# Copyright (c) 2006, Laurent Montel, <montel@kde.org>
```

Comment:
Sorry, I can only accept pull requests written by you and if you accept the license in the top directory.
(Since this does not need more logic than standard cmake find scripts it should be easy to write.)